### PR TITLE
Add user survey to Readme

### PR DESCRIPTION
**Pull request by ebugden**
Having the survey available and visible at the moment prospective users
are trying the tool makes users more likely to give spontaneous
feedback. This feedback can be processed periodically.

Advantages:
 * Feedback is more accurate since users write their impressions right
    after they experience the tool.
 * Source of feedback doesn't require devs having to actively seek it.

Disadvantages:
 * Feedback may take a while to be seen and considered which could make
    some prospective users feel like they're speaking into the void.
    The delay will be especially significant in these early stages
    while the dev process shifts towards receiving continuous feedback.
 * Possible garbage survey results from trolls. This risk seems minimal.

Signed-off-by: Erica Bugden <erica.bugden@gmail.com>

image

### CODE REVIEW

**Comment by ebugden** [comment]
Not sure if the phrasing is endearing or just cringe/confusing. Let me know what you think!

Also whether the survey should be more visible e.g. in it's own section

**Comment by ebugden** [comment]
@ssmagula @MatthewKhouzam @bhufmann Is there still interest in this PR? If not, I will close it.

**Comment by ssmagula** [comment]
This is a great way to get feedback, early and often. Erica, Matthew, and I all have access to the survey results. @MatthewKhouzam do you want me to transfer ownership of the survey to you?

**Comment by MatthewKhouzam** [comment]
Please do so!

**Comment by bhufmann** [comment]
> 

@ssmagula could you please transfer the ownership as suggested?

**Comment by ssmagula** [comment]
Ok, I just sent the invitation to Matthew's gmail. Once he accepts, he'll be the owner of this Trace Viewer User Happiness survey. May it lead to ever increasing user happiness!

**Comment by hoangphamEclipse** [comment]
@bhufmann @MatthewKhouzam Can this PR be closed?

**Comment by MatthewKhouzam** [comment]
Yes!

**Review by MatthewKhouzam — APPROVED** [review]

**Review by marco-miller — CHANGES_REQUESTED** [review]
@ebugden, this is just a friendly flag for you to please sign the CLA per CI's feedback. We may then rebase+merge this PR of yours once that gets green.

**Comment by marco-miller** [comment]
> Hello! I'm assuming this is a new requirement that popped up since I first submitted this. Could someone with an Eclipse account send me a screenshot of the error details and a copy of the CLA? @marco-miller would you be willing to share this information? When I click on details it asks me to create an Eclipse account right away and I would like to read and understand what they're asking me to sign before I decide to create an account.

Sure! The CI failure link points to [1] below. That in turn indirectly points to [2], leading to [3] (the actual agreement). There are also [4,5] as some more potential references. Hope this helps! :)

[1] https://accounts.eclipse.org/legal/eca/validation/137874
[2] https://www.eclipse.org/projects/handbook/#contributing-eca
[3] https://www.eclipse.org/legal/ECA.php
[4] https://www.eclipse.org/legal/ecafaq.php
[5] https://wiki.eclipse.org/Development_Resources/Contributing_via_Git#Eclipse_Contributor_Agreement

**Comment by ebugden** [comment]
Thank you for the helpful and detailed answer @marco-miller! After a quick glance, I think these links will satisfy my curiosity. I plan to take a close look and update the PR within a week or so.

**Comment by ebugden** [comment]
In the end I've decided not to sign the CLA because I'm not comfortable giving Eclipse my personal mailing address. But I am comfortable with you guys copying the line of code (I don't know if it's possible to cherry pick one commit and then change the committer/signoff information to someone who has signed the CLA) or committing functionally equivalent code though if you'd like. Either way, I'm ok with this PR being closed.

**Comment by bhufmann** [comment]
> In the end I've decided not to sign the CLA because I'm not comfortable giving Eclipse my personal mailing address. But I am comfortable with you guys copying the line of code (I don't know if it's possible to cherry pick one commit and then change the committer/signoff information to someone who has signed the CLA) or committing functionally equivalent code though if you'd like. Either way, I'm ok with this PR being closed and similar changes being contributed by someone else.

Thanks for the update. I will close this PR since it cannot be merged as is. We'll take it from there and provide the survey when and where needed.

**Pull request closed without merging**
